skimage/filters/thresholding.py

```diff
@@ -322,9 +322,10 @@ def threshold_otsu(image=None, nbins=256, *, hist=None):
     The input image must be grayscale.
     """
     if image is not None and image.ndim > 2 and image.shape[-1] in (3, 4):
-        msg = "threshold_otsu is expected to work correctly only for " \
-              "grayscale images; image shape {0} looks like an RGB image"
-        warn(msg.format(image.shape))
+        msg = (f"threshold_otsu is expected to work correctly only for "
+               f"grayscale images; image shape {image.shape} looks like "
+               f"an RGB image")
+        warn(msg)
```

Comment:
Is `msg` reused later on?

Reply:
Nope. This felt kind of silly to me as well. I went ahead and got rid of the intermediate variable assignment here for the two you commented on that just go straight to `warn` since it's pretty much the same level of indenting as well, so it doesn't really add anything for readability either